components/TextFields/BUILD

```diff
@@ -78,6 +79,15 @@ mdc_extension_objc_library(
     ],
 )
 
+mdc_examples_objc_library(
+    name = "ObjcExamples",
+    deps = [
+        ":TextFields"
```

Comment:
Missing a comma here.

```
ERROR: /Volumes/BuildData/tmpfs/src/github/repo/components/TextFields/BUILD:85:22: Implicit string concatenation is forbidden, use the + operator
ERROR: /Volumes/BuildData/tmpfs/src/github/repo/components/TextFields/BUILD:86:9: syntax error at '":ColorThemer"': expected ',', 'for' or ']'
ERROR: /Volumes/BuildData/tmpfs/src/github/repo/components/TextFields/BUILD:84:12: Traceback (most recent call last):
	File "/Volumes/BuildData/tmpfs/src/github/repo/components/TextFields/BUILD", line 82
		mdc_examples_objc_library(name = "ObjcExamples", deps = $err...)
	File "/Volumes/BuildData/tmpfs/src/github/repo/components/TextFields/BUILD", line 84, in mdc_examples_objc_library
		$error$
contains syntax error(s)
ERROR: error loading package 'components/TextFields': Package 'components/TextFields' contains errors
```

Reply:
If you would like to test locally (faster than waiting on kokoro):

```
./.kokoro -d bazel -t components/TextFields/...
```

Reply:
Whoops! Thanks Rob